### ISSUE TITLE
Prove Parameterized Representability =~ NatIso

### ISSUE THREAD

**Issue by maxsnew** [maxsnew/multi-poly-cats#7]
Prove the equivalence between the 4 notions in `Cubical.Categories.Profunctor.General`

**Comment by GenericMonkey** [comment]
Looking for an efficient way to do this -- I think ideally we'd want `Functor→Prof*-o`, `Functor→Profo-*` etc to be functors themselves, that way we can better reason about the `NatIso`s in definitions 1 and 2 (in fact, I think the equivalence in one direction is just that the functor preserves `NatIso`s). But maybe defining these as functors is extra work that we don't need to do and all we really need are constructive proofs
```
    ProfRepresentation→PshFunctorRepresentation (G , η) = (G , {!   !})
```

Which way do you think is better?

**Comment by maxsnew** [comment]
While it is a functor (a fully faithful one) I don't see where that comes up in this proof

**Comment by maxsnew** [comment]
Isomorphism is fine, doesn't it imply path-equality?

**Comment by stschaef** [comment]
Closed by #19
